geth/node/status_node.go

```diff
@@ -348,20 +349,36 @@ func (n *StatusNode) gethService(serviceInstance interface{}) error {
 	}
 
 	if err := n.gethNode.Service(serviceInstance); err != nil {
-		return fmt.Errorf("service unavailable: %v", err)
+		return err
 	}
 
 	return nil
 }
 
 // LightEthereumService exposes reference to LES service running on top of the node
 func (n *StatusNode) LightEthereumService() (l *les.LightEthereum, err error) {
-	return l, n.gethService(&l)
+	err = n.gethService(&l)
+	switch err {
+	case node.ErrServiceUnknown:
+		err = ErrServiceUnknown
```

Comment:
Redundant?

Reply:
`node.ErrServiceUnknown` vs `ErrServiceUnknown`. I did it to avoid importing `status-im/status-go/geth/node` and `ethereum/go-ethereum/node`.

Reply:
Ah, thanks, should have read more carefully.